### ISSUE TITLE
Too many redirects on git-lfs.com

### ISSUE THREAD

**Issue by daxmc99** [git-lfs/git-lfs#5204]
**Describe the bug**

Too many redirects on git-lfs.com

**To Reproduce**
1. `curl  -L  "git-lfs.com"`
2. See `curl: (47) Maximum (50) redirects followed`

**Expected behavior**

Able to view the top Google result for git-lfs

**Comment by MadVikingGod** [comment]
To add a bit more info for this.

https://git-lfs.github.com redirects to http://git-lfs.com (no https)
http[s]://git-lfs.com redirects to https://git-lfs.github.com

**Comment by chrisd8088** [comment]
Hey, thanks for the report.  This looks to be related to git-lfs/git-lfs.github.com@5dd037f3a5fd4f4edcf12dd90666646692cd8e7d, which was probably only made live today when the `main` branch was updated with git-lfs/git-lfs.github.com@054bcc0f1af15e93759a58d5c61d082214f12235 and published.

For the time being I've reverted the change to the `CNAME` file in git-lfs/git-lfs.github.com@c09c07bceecee5aa386b455c9fd4d09a903222d3 and that seems to have resolved the immediate problem.
